src/GafferArnoldModule/GafferArnoldModule.cpp

```diff
@@ -95,8 +76,6 @@ BOOST_PYTHON_MODULE( _GafferArnold )
 		)
```

Comment:
Do we still need the binding for loadShader above? Can we just rely on the base class binding instead?